README.md

```diff
@@ -342,6 +342,7 @@ Currently, the following report formats are supported (reporter names are case-i
 * [NOTICE](http://www.apache.org/dev/licensing-howto.html) file in two variants
   * List license texts and copyrights by package (`-f NoticeByPackage`)
   * Summarize all license texts and copyrights (`-f NoticeSummary`)
+* [SPDX Document, version 2.2](https://spdx.dev/specifications/) (`-f SpdxDocument`)
```

Comment:
Now that I looked at the rich diff it could be that pulling the ", version 2.2" part out of the link looks better, like:

    * [SPDX Document](https://spdx.dev/specifications/), version 2.2 (`-f SpdxDocument`)

I'll leave it up to you to decide and will approve anyway.

Reply:
Ah, I read it too late...